docassemble/MassAppealsCourt/data/questions/appeals-basic-questions.yml

```diff
@@ -65,7 +65,7 @@ question: |
   % if email_success:
   Your form is on its way.
   % else:
-  Something went wrong delivering your form
+  Something went wrong delivering your form.
```

Comment:
Do we prefer periods on headers / page title text that is larger than others? I'd lean towards not having periods, but not sure what the rest of the interviews have been doing, and can't find anything about it in the Plain language docs.

Reply:
Changed back in new commit. You're definitely right! I did not realize it was in the question.